index.bs

```diff
@@ -3630,7 +3630,7 @@ An [=aggregatable report=] |report|'s <dfn for="aggregatable report">shared info
 of running the following steps:
 
 1. Let |reportingOrigin| be |report|'s [=aggregatable report/reporting origin=].
-1. Let |sharedInfo| be an [=ordered map=] of the following key/value pairs:
+1. Let |sharedInfo| be a [=map=] of the following key/value pairs:
```

Comment:
Would this affect the comparison of some encrypted value?

Reply:
It wouldn't *affect* anything, since a map *is* an ordered map, but as for whether the ordered nature of the map here is important, it depends on whether https://infra.spec.whatwg.org/#serialize-an-infra-value-to-a-json-string preserves ordering.

Reply:
I meant like for aggregation service, but I think they ignore the field order.

Reply:
@linnan-github Is the order of the fields relevant here?

Reply:
The order of these fields are irrelevant and is not impacting privacy.